src/Test.Utilities/AdditionalMetadataReferences.cs

```diff
@@ -26,6 +26,9 @@ public static class AdditionalMetadataReferences
         public static ReferenceAssemblies DefaultWithNewtonsoftJson { get; } = Default
             .AddPackages(ImmutableArray.Create(new PackageIdentity("Newtonsoft.Json", "10.0.1")));
 
+        public static ReferenceAssemblies DefaultWithSystemWebAndWinForms { get; } = Default
+            .AddAssemblies(ImmutableArray.Create("System.Windows.Forms", "System.Web"));
```

Comment:
💡 Can we try restricting the new one to `DefaultWithWindowsForms`, and using the existing `DefaultWithSystemWeb` for the others? If one test requires both then I guess it's weird but fine.